app/scripts/lib/auth-errors.js

```diff
@@ -269,9 +269,13 @@ define(function (require, exports, module) {
       errno: 1036,
       message: t('Missing property in resume token: %(property)s')
     },
-    INVALID_DATA_FLOW_BEGIN_ATTR: {
+    INVALID_DATA_ATTRIBUTE: {
       errno: 1037,
-      message: t('Invalid data-flow-begin attribute')
+      message: t('Invalid data attribute: %(property)s')
+    },
+    MISSING_DATA_ATTRIBUTE: {
+      errno: 1038,
+      message: t('Missing data attribute: %(property)s')
     }
```

Comment:
Since this uses interpolated messages, I think you may need to add entries to [`toInterpolationContext`](https://github.com/mozilla/fxa-content-server/blob/b2ef9c18f85a039d9e49f247225247553c7c923e/app/scripts/lib/auth-errors.js#L286) to ensure errors are interpolated. Requiring entries in `toInterpolationContext` is set up seems like and anti-pattern to me, it seems like by default, the error itself should be the interpolation context. 

Opened https://github.com/mozilla/fxa-content-server/issues/3784